mod_auth/forms.py

```diff
@@ -5,8 +5,7 @@
 
 from flask_wtf import FlaskForm
 from wtforms import PasswordField, SelectField, StringField, SubmitField
-from wtforms.fields.html5 import EmailField
-from wtforms.fields.simple import PasswordField
+from wtforms.fields import EmailField, PasswordField
```

Comment:
This is still unrelated to this specific PR...

Reply:
Sir, without this the development server was not starting due to WTForms 3.0.0, but okay I will remove this from this PR.

Reply:
As I stated before, it's unrelevant to this PR's changes. If I just merge this it'll most likely break the running platform, which is still on WTForms 2.3.3.